website/blog/2024-02-20-release-2024-2-is-here/item.md

```diff
@@ -0,0 +1,102 @@
+
+## Sure, there are some breaking changes
+
+For the first release of the year, we purposefully included several changes that are required for future product growth, some of which might require manual changes to your system configurations. Be sure to refer to the [Release Notes](https://goauthentik.io/docs/releases/2024.2) for instructions about any actions you might need to take. In general, we strive to reduce the number of breaking changes, and to always be transparent about the actions needed and the reasons for the changes. Please be sure to reach out to us via email to hello@goauthentik.io, or in [GitHub](https://github.com/goauthentik/authentik) or [Discord](https://discord.com/channels/809154715984199690/809154716507963434) with any questions and feedback!
+
+### Tenants renamed to brands
+
+Perhaps the biggest change is that _tenants_ (which were previously used to change branding configuration, default flows, and several other settings) have been renamed to *brands*. The term "Brands" more accurately reflect their usage; to configure branding, logos, colors, and overall login flow behavior. For more information, refer to the [Release Notes](https://goauthentik.io/docs/releases/2024.2) and to our documentation for _brands_.
+
+### **Helm chart breaking changes**
+
+The Helm Chart used for running authentik in Kubernetes has a number of breaking changes. Find out more in the [chart release notes](https://github.com/goauthentik/helm/releases/tag/authentik-2024.2.0).
+
+## More new functionality, DX, and UX enhancements
+
+Release 2024.2 contains even more; the following additional enhancements will make running and operating authentik even more valuable, no matter your role.
+
+-   ### AWS S3 file storage
+
+    Media files can now be stored on AWS S3. To learn more and get started with using S3, follow the [setup guide](https://goauthentik.io/docs/installation/storage-s3).
+
+-   ### *Pretend user exists* option for Identification stage
+
+    Previously the Identification stage would only continue if a user matching the user identifier exists. While this was the intended functionality, this release adds an option to continue to the next stage even if no matching user was found. "Pretend" users cannot authenticate nor receive emails, and don't exist in the database.
+
+-   ### Flows can now be restricted to outposts
+
+    You can now configure a flow to restrict its use, using the flow’s **Require Outpost** setting. This capability is mainly used for LDAP flows.
+
+-   ### Admin can see all of a user’s apps
+
+    A new tab on the **User** details page now displays all applications to which the user has access. From the list of applications, the admin can drill down into a specific app and see details about the app, including a changelog on app modifications, any bindings, and the permissions on the app.
```

Comment:
Typo?  "... on ..."  should maybe read: "... a changelog of app modifications..."